lumispy/signals/luminescence_spectrum.py

```diff
@@ -107,10 +111,20 @@ def to_eV(self, inplace=True, jacobian=True):
                     self.axes_manager.signal_axes[0],
                     evaxis.axis,
                 )
+                if self.metadata.has_item("Signal.Noise_properties.variance"):
```

Comment:
Here, I wonder what happens in case variance is set using the `noise_properties.variance_linear_model` and not the signal object

Reply:
I suppose we should first run the `s.estimate_poissonian_noise_variance()` which would return a signal object for the metadata. Then the function would proceed correctly. Am I correct? Maybe worth including this case too.